src/transformers/models/nllb/tokenization_nllb.py

```diff
@@ -137,10 +137,14 @@ def __init__(
         src_lang=None,
         tgt_lang=None,
         sp_model_kwargs: Optional[Dict[str, Any]] = None,
-        additional_special_tokens=None,
+        additional_special_tokens=FAIRSEQ_LANGUAGE_CODES,
```

Comment:
How are we going to handle the `<mask>` token?
Currently, it is added to the vocabulary after all the language code tokens.

Reply:
The mask token will be added before, that's why we have the failing test! Best would be to either not pass default list of additional special tokens, or just add the mask token to the `self._added_tokens_decoder` list that we enforce (which is what I did for other models)